modules/Core/includes/endpoints/VerifyDiscordEndpoint.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @param string $token The token of the user to update
+ * @param int $discord_id The user's Discord user ID to set
+ * 
+ * @return string JSON Array
+ */
+class VerifyDiscordEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'verifyDiscord';
+        $this->_module = 'Core';
+        $this->_description = 'Verify and link a NamelessMC user\'s Discord account using their validation token';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.